lib/score.rb

```diff
@@ -0,0 +1,66 @@
+require_relative 'scorecard'
+require_relative 'round'
+
+class Score 
+
+    attr_reader :first_bowl, :second_bowl, :total_score, :score_card
+
+    def initialize
+        @score_card = []
+        @total_score = 0
+    end
+
+    def total(x, y)
+    @score_card << [x, y]
+    if @score_card.length > 1
```

Comment:
this is good and useful logic (in the lines below), but it isn't immediately apparent what it is doing because of all the indexes (always a problem with arrays) - is there a possibility of maybe assigning some of these values to variables with clear names, or would that clutter things up too much?